sqlglot/parser.py

```diff
@@ -77,6 +77,7 @@ class Parser(metaclass=_Parser):
 
     FUNCTIONS: t.Dict[str, t.Callable] = {
         **{name: f.from_arg_list for f in exp.ALL_FUNCTIONS for name in f.sql_names()},
+        "APPROX_COUNT_DISTINCT": exp.ApproxDistinct.from_arg_list,
```

Comment:
you can just put this in the sql names in expression

Reply:
Ah right

Reply:
Should I also add `IFNULL` as an alternative name for `Coalesce` while I'm at it?